Jenkinsfile

```diff
@@ -279,6 +280,14 @@ if (env.CHANGE_TITLE && env.CHANGE_TITLE.contains('[ci-skip]')) {
                                                 checkout scm
                                             }
 
+                                        println "Test React Native"
+                                        dir("react-native") {
+                                            sh "npm i"
+                                            sh "npm run gobuild-ios"
+                                            sh "(npm run start &) ; npm run test-ios"
+                                            sh "killall 'Simulator'"
```

Comment:
Not sure, but also, backgrounding `npm run start` will cause a race, if `npm run test-ios` has a dependency on something that `npm run start` does.